docs/source/fractal-results.rst

```diff
@@ -0,0 +1,94 @@
+Results
+========
+
+Results are a single quantum chemistry method evaluation including energy,
+gradient, Hessian, and property evaluations. Collections of evaluations such
+as finite-difference gradients, complete basis set extrapolation, or geometry
+optimizations belong under the "Procedures" heading.
```

Comment:
"A result is a single quantum chemistry method evaluation, which might be an energy, an analytic gradient or Hessian, or a property evaluation."